### ISSUE TITLE
NPE in FeadbackActivity v5.0.1

### ISSUE THREAD

**Issue by Khang-NT** [bitstadium/HockeySDK-Android#307]
```
java.lang.NullPointerException: Attempt to invoke interface method 'boolean java.util.List.removeAll(java.util.Collection)' on a null object reference
	at net.hockeyapp.android.FeedbackActivity$FeedbackHandler.handleMessage(FeedbackActivity.java:849)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:6872)
	at java.lang.reflect.Method.invoke(Method.java)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1404)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1199)
```

**Comment by Khang-NT** [comment]
I saw that bug already fixed on commit https://github.com/bitstadium/HockeySDK-Android/commit/6589c87e8549b392422522762c7eba7c76c8a152 
Can you guys publish new version include that change?

**Comment by TroubleMakerBen** [comment]
Hey @Khang-NT,

we just released 5.0.2 of the SDK to fix this bug.

Thanks for letting us know and for using HockeyApp!
Cheers!
Benny
